WalletWasabi.Tests/UnitTests/Tor/Control/TorControlClientTests.cs

```diff
@@ -43,13 +43,13 @@ public async Task ReceiveTorAsyncEventsUsingForeachAsync()
 						break;
 					}
 
-					await Task.Delay(200).ConfigureAwait(false);
```

Comment:
Please don't do these changes in Tor-related tests. It was very carefully checked for correctness and the reason why `ConfigureAwait(false)` is used is to improve performance in those tests. 

`ConfigureAwait(false)` cannot be in xUnit tests on the top-level only. On line 46, the configure await is wrapped in `Task serverTask = Task.Run(async () =>` and as such one can use CAF(false).

Reply:
Same as before. This is a submethod.